test/test_train_mp_imagenet_torchrun.py

```diff
@@ -0,0 +1,385 @@
+from absl import logging
```

Comment:
to reviewers: this is a torchrun version of test/test_train_mp_imagenet.py. 
Maybe I should merge it with test/test_train_mp_imagenet.py..

Reply:
Are there any differences?

Reply:
Only minor changes. In test_train_mp_imagenet.py, we have
```
if __name__ == '__main__':
  xmp.spawn(_mp_fn, args=(FLAGS,), nprocs=FLAGS.num_cores)
```
This file stops calling `spawn` and changes to:
```
if __name__ == '__main__':
  if not dist.is_torchelastic_launched():
    logging.error('Test must be launched with torchrun!')
    exit(1)
  _mp_fn(FLAGS)
```

Reply:
Can you merge these two cases? ie if the program has been launched with torchrun, call `_mp_fn`, otherwise call `xmp.spawn`

Reply:
done